src/TextField/TextField.stories.tsx

```
@@ -0,0 +1,97 @@
+import { Meta, Story } from "@storybook/react";
+
+import MaskedInput from "react-text-mask";
+import React from "react";
+import TextField from "./TextField";
+import { TextFieldProps } from "./TextField.types";
+import { action } from "@storybook/addon-actions";
+
+const meta: Meta<typeof TextField> = {
+  component: TextField,
+  title: "Text/TextField"
+};
+export default meta;
+
+const Template: Story<TextFieldProps> = args => {
```

Default story not handling masked error if the isFieldMasked is set to true